src/pages/blog/components/post-card/PostCard.tsx

```diff
@@ -0,0 +1,143 @@
+import { Link } from "react-router-dom";
+
+import { Avatar, Box, IconButton, Typography } from "@mui/material";
+import { useMediaQuery } from "@mui/material";
+import { Icon } from "@mui/material";
```

Comment:
Prefer using only default imports from mui, it improves performance

https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
Fixed imports from mui

![image](https://github.com/user-attachments/assets/0e873dcf-8c54-4f17-830f-f449d483c33e)
![image](https://github.com/user-attachments/assets/ee68ddfb-6a09-4147-838e-61fea5479c1f)
![image](https://github.com/user-attachments/assets/6240602d-1665-4c71-bc36-0de9e0d0cf96)